openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -60,32 +60,35 @@ This specification uses the terms "Access Token", "Authorization Endpoint", "Aut
 
-W3C Verifiable Credential:
-:  A Verifiable Credential compliant to the [@VC_DATA] specification.
+Credential Format:
+:  A format used to represent a Verifiable Credential. The exact parameters required to describe a Credential Format are defined in the Credential Format Profile.
+
+Credential Format Profile:
+:  Sets of parameters specific to individual Credential Formats. Credential Format Profiles for W3C Verifiable Credentials [@VC_DATA] and ISO/IEC 18013-5 mDL [@ISO.18013-5] are provided in (#format_profiles). Other specifications or deployments can define their own Credential Format Profiles using the extension points defined in this specification.
+
+Credential Configuration:
+:  A Credential Configuration consists of a Credential Format including a type, information about requesting Credentials using this Credential Configuration (in particular, a `scope` value), information on cryptographic methods and algorithms supported for issuance, and display information to be used by the Wallet. A Credential Configuration is identified by a unique string.
 
 Presentation:
 : Data that is presented to a specific verifier, derived from one or more Verifiable Credentials that can be from the same or different Credential Issuers.
 
 Verifiable Presentation (VP):
 :  A Holder-signed Credential whose integrity can be cryptographically verified to provide Cryptographic Holder Binding. It can be of any format used in the Issuer-Holder-Verifier Model, including, but not limited to those defined in [@VC_DATA] and [@ISO.18013-5].
 
-W3C Verifiable Presentation:
-:  A Verifiable Presentations compliant to the [@VC_DATA] specification.
-
-Credential Issuer:
-:  An entity that issues Verifiable Credentials. Also called Issuer. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Authorization Server (see [@!RFC6749]).
+Credential Issuer (or Issuer):
+:  An entity that issues Verifiable Credentials. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Authorization Server (see [@!RFC6749]).
 
 Holder:
 :  An entity that receives Verifiable Credentials and has control over them to present them to the Verifiers as Verifiable Presentations.
 
 Verifier:
-:  An entity that requests, receives, and validates Verifiable Presentations. During presentation of Credentials, Verifier acts as an OAuth 2.0 Client towards the Wallet that is acting as an OAuth 2.0 Authorization Server. The Verifier is a specific case of an OAuth 2.0 Client, just like Relying Party (RP) in [@OpenID.Core].
+:  An entity that requests, receives, and validates Verifiable Presentations. During a presentation, the Verifier acts as an OAuth 2.0 Client towards the Wallet that is acting as an OAuth 2.0 Authorization Server. The Verifier is a specific case of an OAuth 2.0 Client, just like Relying Party (RP) in [@OpenID.Core].
```

Comment:
> towards the Wallet that is acting as an OAuth 2.0 Authorization Server

Not sure because the Wallet is a client in the issuance, with protocol specific flows and an Identity Provider in the presentation phase (where the data provider was the VCI and the data the VC). The OpenID4VP and SIOPv2 are too much specialized to considering Wallets an AS. Conceptually we can say that a wallet is an AS but I would not define this so explicitly. My thoughs.

Reply:
Perhaps the definition in this spec should be much shorter, just the first sentence. We only need the concept defined in this spec, no further technical details as this is covered by OID4VP.